lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/impl/HandlerContainer.java

```diff
@@ -56,8 +56,13 @@
     this.name = name;
     this.codec = codec;
 
-    LOG.log(Level.FINER, "Instantiated HandlerContainer {0} with codec {1}",
-        new String[] {this.name, this.codec.getClass().getCanonicalName()});
+    LOG.log(Level.FINER, "Instantiated {0}", this.toString());
```

Comment:
nit: if we pass `this` as an argument, Logger.log() does not invoke `toString()` otherwise the log level is configured lower than `FINER`.